include/mbedtls/sha256.h

```diff
@@ -167,12 +175,12 @@ MBEDTLS_DEPRECATED static inline void mbedtls_sha256_update(
 }
 
 /**
- * \brief          SHA-256 final digest
+ * \brief          The SHA-256 final digest.
```

Comment:
Blocker: Please uniformize this with the documentation of the superseding functions.

Reply:
done